src/mpid/ch4/src/mpidig_part_utils.h

```diff
@@ -2,12 +2,38 @@
  * Copyright (C) by Argonne National Laboratory
  *     See COPYRIGHT in top-level directory
  */
-
 #ifndef MPIDIG_PART_UTILS_H_INCLUDED
 #define MPIDIG_PART_UTILS_H_INCLUDED
 
 #include "ch4_impl.h"
 
+/* defines the different stages a send partitions must go through */
+enum MPIDIG_part_status_send_cc_t {
+    MPIDIG_PART_STATUS_SEND_READY,      /* 0 = partition is ready to be sent */
+    /* below insert all the needed operations */
+    MPIDIG_PART_STATUS_SEND_CTS,        /* CTS received */
+    /* the last value is the initialization value */
+    MPIDIG_PART_STATUS_SEND_INIT
+};
```

Comment:
This is a little misleading since reading a value of `1` does not always indicate that CTS has been received.

Reply:
indeed, the goal of that tag is to have the correct value for `INIT`. 
maybe renaming it to `MPIDIG_PART_STATUS_SEND_TMP_1` to avoid any confusion?